docs/application/native/guides/multimedia/openal.md

```diff
@@ -44,6 +44,11 @@ The main features of the OpenAL API include:
 
   > **Note**  
   > The device cannot fall into a sleep state while the context is not destroyed. To avoid unwanted battery consumption, destroy the context punctually.
+  >
+  > Since 3.0, extension alcDevicePauseSOFT() was introduced which allows applications to pause a playback device explictly.
```

Comment:
Please change "Since 3.0, extension alcDevicePauseSOFT() was introduced which allows applications to pause a playback device explictly."
to 
"Since 3.0, extension `alcDevicePauseSOFT()` was introduced to allow applications to pause a playback device explicitly.

Reply:
done

Reply:
There is a mis-typo : explictly -> explic**i**tly